cluster.tf

```diff
@@ -1,104 +1,89 @@
 
 locals {
-  worker_pools_map = module.dynamic_values.worker_pools_map # Convert list to map
-  clusters_map     = module.dynamic_values.clusters_map     # Convert list to map
-  latest_kube_version = {
-    openshift = "${data.ibm_container_cluster_versions.cluster_versions.valid_openshift_versions[length(data.ibm_container_cluster_versions.cluster_versions.valid_openshift_versions) - 1]}_openshift"
-    iks       = data.ibm_container_cluster_versions.cluster_versions.valid_kube_versions[length(data.ibm_container_cluster_versions.cluster_versions.valid_kube_versions) - 1]
-  }
+  cluster_1 = length(module.dynamic_values.clusters_map) == 1 ? module.dynamic_values.clusters_list[0] : null
+  cluster_2 = length(module.dynamic_values.clusters_map) == 2 ? module.dynamic_values.clusters_list[1] : null
 }
 
 ##############################################################################
-
-
-##############################################################################
-# Create IKS/ROKS on VPC Cluster
+# Create ROKS on VPC Cluster
 ##############################################################################
 
-resource "ibm_container_vpc_cluster" "cluster" {
-  for_each          = local.clusters_map
-  name              = "${var.prefix}-${each.value.name}"
-  vpc_id            = each.value.vpc_id
-  resource_group_id = local.resource_groups[each.value.resource_group]
-  flavor            = each.value.machine_type
-  worker_count      = each.value.workers_per_subnet
-  kube_version = (
-    lookup(each.value, "kube_version", null) == "latest" # if version is latest
-    || lookup(each.value, "kube_version", null) == null  # or if version is null
-    ? local.latest_kube_version[each.value.kube_type]    # use latest
-    : each.value.kube_version                            # otherwise use value
-  )
-  update_all_workers = lookup(each.value, "update_all_workers", null)
-  tags               = var.tags
-  wait_till          = var.wait_till
-  entitlement        = each.value.entitlement
-  cos_instance_crn   = each.value.cos_instance_crn
-  pod_subnet         = each.value.pod_subnet
-  service_subnet     = each.value.service_subnet
-
-  dynamic "zones" {
-    for_each = each.value.subnets
-    content {
-      subnet_id = zones.value["id"]
-      name      = zones.value["zone"]
-    }
-  }
 
-  dynamic "kms_config" {
-    for_each = each.value.kms_config == null ? [] : [each.value.kms_config]
-    content {
-      crk_id           = module.key_management.key_map[kms_config.value.crk_name].key_id
-      instance_id      = module.key_management.key_management_guid
-      private_endpoint = kms_config.value.private_endpoint
-    }
+module "cluster_1" {
+  depends_on = [
+    module.vpc, module.observability_instances
+  ]
+  count             = local.cluster_1 != null ? 1 : 0
+  source            = "git::https://github.com/terraform-ibm-modules/terraform-ibm-ocp-all-inclusive.git?ref=v1.0.0"
```

Comment:
Just use the ocp-base module (https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc) for now to simplify - so that we can narrow down the scope of this PR to just consolidating the ocp logic without adding new capabilities. We can add the observability layer in a subsequent step.